README.md

```diff
@@ -13,22 +14,50 @@ docker run --rm \
   -v /tmp/.X11-unix:/tmp/.X11-unix \
   -v /dev/bus/usb:/dev/bus/usb \
   -v /usr/share/X11/xkb:/usr/share/X11/xkb:ro \
-  -v ~/.AndroidStudio:/home/developer/.AndroidStudio \
-  -v ~/.AndroidStudio.android:/home/developer/.android \
   -v ~/.AndroidStudio.java:/home/developer/.java \
-  -v ~/.AndroidStudio.gradle:/home/developer/.gradle \
-  -v ~/.AndroidStudio.sdk:/home/developer/Android/Sdk \
-  -v ~/Project:/home/developer/Project \
-  --name androidstudio-$(head -c 4 /dev/urandom | xxd -p)-$(date +'%Y%m%d-%H%M%S') \
-rycus86/androidstudio:3.4
+  -v ~/.AndroidStudio.config:/home/developer/.config \
```

Comment:
This line makes the initial configuration of the IDE persistent across restarts.
```suggestion
  -v ~/.AndroidStudio.config:/home/developer/.config \
```